testsuite/docs/optional.md

```diff
@@ -127,3 +127,43 @@ Inside of the testsuite, the scenarios that are tagged with
 @nomirror
 ```
 are executed only if you don't use a mirror.
+
+
+### Testing virtualization features
+
+Using a virtualization host with the testsuite is not mandatory.
+
+If you do not want a virtualization host minion, do not define `VIRT_SERVER_KVM` and
+`VIRT_SERVER_XEN` environment variables before you run the testsuite. That's all.
+If you want virtualization minions, make these variables point to the machines that
+will be the virtualization KVM or Xen host minions and define the `VIRT_SERVER_KVM_PASSWORD`
+and `VIRT_SERVER_XEN_PASSWORD` variables:
+
+```bash
+export VIRT_SERVER_KVM=myvirthost.example.com
+export VIRT_SERVER_KVM_PASSWORD=therootpwd
```

Comment:
for consistency, it should be
```
VIRT_SERVER_KVM_URL=myvirthost.example.com
```
you could shorten with
```
VIRTHOST_KVM_URL
VIRTHOST_KVM_PASSWORD
```

Reply:
OK